jobs/ml_jobs/artist_linkage/cli/summarize_biographies_with_llm.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import typer
+
+from src.constants import (
+    ARTIST_ID_KEY,
+    ARTIST_NAME_KEY,
+    ENV_SHORT_NAME,
+    WIKIPEDIA_CONTENT_KEY,
+)
+from src.llm_config import MAX_CONCURRENT_LLM_REQUESTS
+from src.utils.llm import summarize_biographies_with_llm
+
+app = typer.Typer()
+
+
+@app.command()
+def main(
+    artists_with_wikipedia_content: str = typer.Option(),
+    output_file_path: str = typer.Option(),
+    debug: bool = typer.Option(False),
+) -> None:
+    artists_df = pd.read_parquet(artists_with_wikipedia_content)
+
+    # Prepare Data
+    artists_to_summarize_df = artists_df.loc[
+        lambda df: df[WIKIPEDIA_CONTENT_KEY].notna()
+    ].loc[lambda df: df[ARTIST_NAME_KEY].notna()]
+
+    # Predict only on few artists for testing and staging
+    if ENV_SHORT_NAME != "prod":
```

Comment:
Add Optional limit in job and control this from airflow etl instead of depending on env var.